packages/components/.stylelintrc.js

```diff
@@ -0,0 +1,150 @@
+    // 'max-line-length': [200, { ignore: ['non-comments'], ignore: ['comments'], ignorePattern: ["/.*; //.*/"] }],
+    'max-line-length': null,
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://stylelint.io/user-guide/rules/list/number-leading-zero/
+    'number-leading-zero': null,
+
+    // no strong opinions, probably would keep what we have now (no big impact)
+    // see: https://stylelint.io/user-guide/rules/list/number-max-precision/
+    'number-max-precision': 5,
+
+    // I would prefer to be explicit when a mixin is called
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-mixin-argumentless-call-parentheses/README.md
+    'scss/at-mixin-argumentless-call-parentheses': 'always',
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-rule-conditional-no-parentheses/README.md
+    'scss/at-rule-conditional-no-parentheses': null,
+
+    // not sure why we would want this rule active as is by default, probably needs some tweaking (so my suggestion is to simply disable it)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/double-slash-comment-empty-line-before/README.md
+    'scss/double-slash-comment-empty-line-before': null,
+
+    // while in theory this is a good idea, probably I would turn off this rule because it's buggy and triggers an error for us that doesn't make sense
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-unspaced/README.md
+    // (many issues https://github.com/stylelint-scss/stylelint-scss/issues?q=is%3Aissue+is%3Aopen+%22operator-no-unspaced%22)
+    'scss/operator-no-unspaced': null,
+
+    // probably a good idea, but we have the make sure this will not impact consumers (may be using old versions of Sass?)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/no-global-function-names/README.md
+    'scss/no-global-function-names': null,
+
+    // using the default one doesn't work for us (we're using BEM)
+    // found this pattern here: https://github.com/humanmade/coding-standards/pull/199
+    'selector-class-pattern': ['^(?<block>(?:[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)(?<element>(?:__[a-z][a-z0-9]*(?:-[a-z0-9]+)*))?(?<modifier>(?:--[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)?$', { "resolveNestedSelectors": true }],
+
+    // the 'always-multi-line' option works for me
+    // see: https://stylelint.io/user-guide/rules/list/selector-list-comma-newline-after/
+    'selector-list-comma-newline-after': 'always-multi-line',
+
+    // strong-ish option here to disable this check: I prefer being explicit vs implicit (but if there are stronger opinions, I can live with it)
+    // see: https://stylelint.io/user-guide/rules/list/shorthand-property-no-redundant-values/
+    'shorthand-property-no-redundant-values': null,
```

Comment:
I don't see a need to necessarily enforce a rule but repeating the exact same value 4 times such as for a margin for me is overly verbose and harder to read.

IE: I prefer `margin: 1px` vs. `margin: 1px 1px 1px 1px`